-------------------------------------------------------------------------------
Test set: com.example.acceptancetests.AcceptanceTestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 5.601 s <<< FAILURE! - in com.example.acceptancetests.AcceptanceTestSuite
Authentication.Authentication Login #1  Time elapsed: 1.705 s  <<< FAILURE!
org.opentest4j.MultipleFailuresError: 
Multiple Failures (2 failures)
	org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The application at 'D:\TIC\Kampus Merdeka\MSIB\Studi Independen\Alterra\Capstone\AutoMobile\sample-mobile-automation\src\test\resources\application\app-release.apk' does not exist or is not accessible
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'MYLNAP', ip: '192.168.34.45', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'D:\TIC\Kampus Merdeka\MSIB\Studi Independen\Alterra\Capstone\AutoMobile\sample-mobile-automation\src\test\resources\application\app-release.apk' does not exist or is not accessible
    at getResponseForW3CError (D:\user\Mobile\Appium-windows-1.21.0\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (D:\user\Mobile\Appium-windows-1.21.0\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'MYLNAP', ip: '192.168.34.45', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16'
Driver info: driver.version: AndroidDriver
	java.lang.NullPointerException: <no message>

Authentication.Authentication Login #2  Time elapsed: 0.682 s  <<< FAILURE!
org.opentest4j.MultipleFailuresError: 
Multiple Failures (2 failures)
	org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The application at 'D:\TIC\Kampus Merdeka\MSIB\Studi Independen\Alterra\Capstone\AutoMobile\sample-mobile-automation\src\test\resources\application\app-release.apk' does not exist or is not accessible
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'MYLNAP', ip: '192.168.34.45', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'D:\TIC\Kampus Merdeka\MSIB\Studi Independen\Alterra\Capstone\AutoMobile\sample-mobile-automation\src\test\resources\application\app-release.apk' does not exist or is not accessible
    at getResponseForW3CError (D:\user\Mobile\Appium-windows-1.21.0\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (D:\user\Mobile\Appium-windows-1.21.0\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'MYLNAP', ip: '192.168.34.45', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16'
Driver info: driver.version: AndroidDriver
	java.lang.NullPointerException: <no message>

